service/src/main/java/tds/assessment/web/endpoints/AssessmentInfoController.java

```diff
@@ -26,9 +26,15 @@ public AssessmentInfoController(final AssessmentInfoService assessmentInfoServic
         this.assessmentInfoService = assessmentInfoService;
     }
 
-    @GetMapping(value = "/{clientName}/assessments/", produces = MediaType.APPLICATION_JSON_VALUE)
+    @GetMapping(value = "/{clientName}/assessments", produces = MediaType.APPLICATION_JSON_VALUE)
     @ResponseBody
-    ResponseEntity<List<AssessmentInfo>> findAssessment(@PathVariable final String clientName, @RequestParam final String... assessmentKeys) {
+    ResponseEntity<List<AssessmentInfo>> findAssessmentInfos(@PathVariable final String clientName, @RequestParam final String... assessmentKeys) {
         return ResponseEntity.ok(assessmentInfoService.findAssessmentInfo(clientName, assessmentKeys));
     }
+
+    @GetMapping(value = "/{clientName}/assessments/grade/{grade}", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Why have a separate endpoint for this since it returns the same resource?  It seems like the existing endpoint should have an optional query parameter that will narrow it by grade.

`/SBAC_PT/assessments?grade=3`